fapolicy_analyzer/ui/remove_deleted_dialog.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Concurrent Technologies Corporation 2021
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from fapolicy_analyzer.ui.ui_widget import UIBuilderWidget
+from fapolicy_analyzer.util.format import f
+from locale import gettext as _
+
+
+class RemoveDeletedDialog(UIBuilderWidget):
+    def __init__(self, deleted=[], parent=None):
+        def plural(count):
+            return ("s", "are") if count > 1 else ("", "is")
+
+        super().__init__()
+        if parent:
+            self.get_ref().set_transient_for(parent)
+
+        textView = self.get_object("removeInfo")
+        textBuffer = textView.get_buffer()
+
+        displayText = f(_("{len(deleted)} file{plural(len(deleted))[0]} cannot be found on disc"))
```

Comment:
"cannot be found on the file system" would be more consistent with the working in the file details panes.

Reply:
If you are keeping the current text at least change the spelling to "disk".